base_classes/NXdata.nxdl.xml

```diff
@@ -309,6 +309,20 @@
 
 		</doc>
 	</attribute>
+    <attribute name="reference">
+        <doc>
+             Points to the path of a field defining the data to which the `DATA` group refers.
+
+             This concept allows to link the data to a respective field in the NeXus hierarchy, thereby
```

Comment:
suggest rewording to avoid using the word "link":
"This attribute indicates the origin of the data, thereby providing an association to further metadata that define the physical quantity that the data represents"